src/test/java/htsjdk/samtools/SAMReadGroupRecordTest.java

```diff
@@ -145,4 +148,25 @@ public void testEqualsAndHashcode(final SAMReadGroupRecord rg, final Object othe
         }
     }
 
+    @DataProvider
+    public Object[][] getBarcodes(){
```

Comment:
missing space before `{` on a few lines here
```suggestion
    public Object[][] getBarcodes() {
```